common/models/model-method.json

```diff
@@ -18,9 +18,6 @@
     "aliases": {
       "type": "array"
     },
-    "isStatic": {
-      "type": "boolean"
-    },
```

Comment:
I think we should keep this property around, so that consumers of loopback-workspace do not need to understand how exactly are remote methods described in JSON files. That way the API of loopback-workspace remains the same, which means less work for us too.